### PR TITLE
Update Polish translation

### PR DESCRIPTION
**Pull request by ArturWroblewski**
4 translations added

### CODE REVIEW

File: data/po/pl.po [hunk 1]
```diff
@@ -243,10 +243,9 @@ msgstr "Pokaż FPS w grach:"
 
 #. Has to end with ": "
 #: data/ui/preferences.ui:201
-#, fuzzy
 msgctxt "show_windows_games"
 msgid "Show Windows games: "
-msgstr "Pokaż FPS w grach:"
+msgstr "Pokaż FPS w grach: "
```
Comment (sharkwouter) [inline data/po/pl.po:248]: I think this should be changed. It still contains FPS

**Comment by sharkwouter** [comment]
Thanks for the PR! It looks good, but I did find one small issue

File: data/po/pl.po (hunk 1 above)
Comment (ArturWroblewski) [reply data/po/pl.po:248]: You're right . I don't know how it could get there :)

**Comment by ArturWroblewski** [comment]
> Thanks for the PR! It looks good, but I did find one small issue

Fixed

**Comment by sharkwouter** [comment]
Thanks! I've merged it

**Pull request merged**